### ISSUE TITLE
Runway Indicator and Trend Localization Problem

### ISSUE THREAD

**Issue by Suheyb** [mivek/MetarParser#320]
Hi,
While I using RunwayInfo data type, I came accross language problem at 'trend' and 'indicator'. I think before parsing metar message which localization chosen, trend and indicator will be at this langugage. Maybe, indicator and trend can be Enum type like DepositType, DepositCoverage etc. Is there any other solution for this problem?
Thanks

**Comment by mivek** [comment]
Hello @Suheyb 
Can you please provide an example of parsing: the output given by the parser and the expected result?

**Comment by Suheyb** [comment]
```
    private static void translateExample() {
        Messages.getInstance().setLocale(new Locale("", ""));
        Metar metar = MetarParser.getInstance().parse("LTDA 200150Z 00000KT 0600 0200SW R22/P1500U R04/0275D BCFG VV006 03/03 Q1023 RMK RWY04 29003KT 250V330 =");
        metar.getRunways().forEach(rInfo -> System.out.println(getMessage("PR.rwy.runway",rInfo.getName()) + " Trend : "+ rInfo.getTrend() + " Indicator : " + rInfo.getIndicator()));
        Messages.getInstance().setLocale(new Locale("en", "US"));
        metar.getRunways().forEach(rInfo -> System.out.println(getMessage("PR.rwy.runway",rInfo.getName()) + " Trend : "+ rInfo.getTrend() + " Indicator : " + rInfo.getIndicator()));
    }
```

And the output of this code block is :
Pist 22  Trend : artan Indicator : daha fazla 
Pist 04  Trend : azalan Indicator : 
Runway 22  Trend : artan Indicator : daha fazla 
Runway 04  Trend : azalan Indicator : 

My default messages.properties file in Turkish language. As you see  PR.rwy.runway's value change, when locale changes but Trend and Indicator doesn't change.

**Comment by mivek** [comment]
Hello @Suheyb 

This is the normal behaviour, when you parse the metar, the locale was `new Locale("","")` defaulting to the locale of your computer. 
The attribute `RunwayInfo.trend` is a string, the value cannot be changed by changing the locale. 
This is the same behaviour for the remarks.

**Comment by Suheyb** [comment]
Hello @mivek 
I know String cannot change, when locale change. Do you have plan for change this behaviour?

**Comment by mivek** [comment]
@Suheyb I think changing the trend to an enum should not be too complicated. I will check which other attributes can be migrated to enum. For the moment I see:
- RunwayInfo.trend
- RunwayInfo.indicator

Also, if you want to add Turkish translation to the project, you can do it [here](https://crwd.in/metarParser)

**Comment by Suheyb** [comment]
@mivek I am happy to contribute to the project but I think you should add Turkish language option.

**Comment by mivek** [comment]
I added the language, it should be ok now

**Comment by mivek** [comment]
Hello @Suheyb 

I saw on crowdin that you completed Turkish locale to 28%. I wanted to add Turkish locale in #321 . Should I wait more to let you complete the Turlisk locale?

**Comment by Suheyb** [comment]
Hi @mivek 

I added to crowdin what I translate. I can translate others but I can't guaraante all of them will be true.  If it is ok, I can complete translation in 1 week.

**Comment by mivek** [comment]
Ok then I'll merge the opened pull request and create another one for Turkish locale.

**Comment by mivek** [comment]
@Suheyb 

I merged the PR. The new feature is in version 2.8.0 of the project.

**Comment by Suheyb** [comment]
@mivek I almost complete translation. Only I want to know what's the meaning of SM?

**Comment by mivek** [comment]
Hello @Suheyb 

SM is a unit meaning statute miles. This unit is often used in US METARs.

1 SM = 1609,3 meters

**Comment by Suheyb** [comment]
Hi @mivek 
I completed translation. I'm happy to contribute this project. Thanks for your work.

**Comment by mivek** [comment]
Thanks a lot @Suheyb 
I will integrate the translation in the next release

**Comment by mivek** [comment]
Hello @Suheyb 

The Turkish translation is available in version 2.8.1 of the project.
Thank you again for your help

**Comment by Suheyb** [comment]
Hi @mivek,

I will import new version. Thanks a lot for your work.
